stdlib/public/runtime/ImageInspection.h

```diff
@@ -30,6 +30,18 @@ namespace swift {
     void *symbolAddress;
   };
 
+#if defined(__ELF__) || defined(__ANDROID__)
```

Comment:
Could we separate this #if block out into a separate `ImageInspectionELF.h` header?